hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -808,16 +808,12 @@ public void restoreToInstant(final String instantTime) throws HoodieRollbackExce
             instantsToStats.put(instant.getTimestamp(), statsForInstant);
             break;
           case HoodieTimeline.COMPACTION_ACTION:
-            if (instant.isRequested()) {
-              // TODO : Get file status and create a rollback stat and file
-              // TODO : Delete the .aux files along with the instant file, okay for now since the archival process will
-              // delete these files when it does not see a corresponding instant file under .hoodie
-              deleteRequestedCompaction(instant.getTimestamp());
```

Comment:
Can we remove this method altogether ?

Reply:
Let's keep it for now, need to do more clean up around this.